airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py

```diff
@@ -549,3 +549,18 @@ def test_cursorless_incremental_substream(requests_mock, bank_accounts, sync_mod
     for slice_ in stream.stream_slices(sync_mode=sync_mode, stream_state=stream_state):
         for record in stream.read_records(sync_mode=sync_mode, stream_state=stream_state, stream_slice=slice_):
             stream.get_updated_state(stream_state, record)
+
+
+@pytest.mark.parametrize("stream", ("bank_accounts",))
+def test_get_updated_state(stream, request, requests_mock):
```

Comment:
Should we have a test that validates that if a record does not have `cursor_field` and `legacy_cursor_field`, we use `now` as a `cursor_field`?